src/serialization/toSerializableFormat.js

```diff
@@ -32,15 +32,18 @@ function saveStatus(originalState, serializableState) {
  * @returns {*}
  */
 function transformSubstate(originalSubState) {
-  if (_.isPlainObject(originalSubState)) {
-    // eslint-disable-next-line  no-use-before-define
-    return toSerializableFormat(originalSubState);
-  } else if (_.isArray(originalSubState) && originalSubState[STATUS]) {
+  if (_.isArray(originalSubState)) {
+    if (!originalSubState[STATUS]) {
```

Comment:
I still do not understand, you can have the condition in one line`_.isArray(originalSubState) && originalSubState[STATUS]`? If this is not true, `originalSubState` is going to be returned.

`_. isPlainObject([])` will return `false`.